package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tests-selector",
-  "version": "1.0.0",
+  "version": "1.1.3",
```

Comment:
Please change 1.1.3 -> 1.1.4 so I can publish this version

Reply:
in package-lock ? maybe it would be better to remove package-lock ?

Reply:
My bad. I thought it was package. But package version still has to be upgraded

Reply:
@Gennadiii i think you will do it before publishing ?

Reply:
Yeah I will